### ISSUE TITLE
Delimiters and keyCode limitation, with non-English keyboards

### ISSUE THREAD

**Issue by ajmas** [i-like-robots/react-tags#87]
There appears to be a small limitation when using keyCode as delimiter. Using the comma, as example:
```
   ','.keyCode(0) --> 44
   comma key pressed returns 188
```
So the solution would appears to be use use `delimiters=[ 188 ]`, but this actually has a limitation if we are targeting a user base where English is not the UI language and the keyboard places the comma elsewhere, such as with the Arabic keyboard. In this scenario the keyCode is that of the 'K' key.

Testing suggest the `event.key` value is actually more accurate in this case. To allow for the character, on top of the key code, while maintaining backwards compatibility, the simplest approach could be a new property:
```
   delimterChars=[',']
```
So the resultant code would be:
```
if (this.props.delimiters.indexOf(e.keyCode) !== -1 || this.props.delimitersChars.indexOf(e.key)) { ... }
```
If you think this is an acceptable solution I could create a PR for this?

**Comment by i-like-robots** [comment]
Hi @ajmas, thank you for taking the time to work on this. I reckon this addresses the same issue as #84 (might be worth passing your eyes over the thread). This looks to be a simple solution without breaking compatibility (`event.key` has been discussed in #81, too). I'll give it a test when I get a moment.

**Comment by ajmas** [comment]
For issue #84 is slightly different, since that would need to depend on the `onPaste` event, but it could benefit from the delimiterChars array.

**Comment by i-like-robots** [comment]
Yes, sorry - I meant if we could use the same option to tackle both issues. Or even replacing the current keyCode option with an array of characters... though that may get confusing if a user intentionally wishes to type the word `Enter` or `Backspace`!

**Comment by i-like-robots** [comment]
Fixed with #88 and #89  which will be in the 5.5.0 release
